docs/source/install-docs/Physical-NodeInstall-20.04

```diff
@@ -0,0 +1,219 @@
+        3. Also have them send you their Public IP address so that you can add it to the firewall to allow them access. Optionally, have them send a preferred username also.
+    2. Add their IP addresses to the firewall:
+    3. Add the users to the server:
+        1. Login to the Node as the base user.
+        2. Run the following commands, substituting the username in for &lt;newuser>
+        3. `sudo adduser &lt;newuser>`
+            1. You can safely ignore messages like “sent invalidate(passwd) request, exiting“
+            2. For “Enter new UNIX password:”  put password1 (This will be changed later)
+            3. Enter a name (optional)
+            4. Defaults are fine for the rest
+        4. `sudo usermod -aG sudo &lt;newuser>`
+        5. Then create a file in the newusers home directory:
+            1. `sudo mkdir /home/&lt;newuser>/.ssh` 
+            2. `sudo chown &lt;newuser>:&lt;newuser> /home/&lt;newuser>/.ssh`
+            3. `sudo vim /home/&lt;newuser>/.ssh/authorized_keys`
+            4. Paste the users public key into the open file and then save it (:wq)
+            5. `sudo chown &lt;newuser>:&lt;newuser> /home/&lt;newuser>/.ssh/authorized_keys`
+        6. Repeat the above for each new admin user you create.
+    4. The new users are now able to login. Since 2FA is required, when you send the password to each of the new users, also send the following instructions (HINT: fill in the username, Client IP address, and password for them with the correct values):
+        1. Thanks for agreeing to help with the administration of our Indy Validator Node. Please login to the node, change your password, and setup Two Factor Authentication (2FA) using the following instructions:
+            1. ssh -i &lt;your private SSH key file> &lt;username>@&lt;Client IP Addr>
+            2. Type in password1 for your password
+            3. On successful login, type in ‘passwd’ to change your password on the Validator Node. Please use a unique password of sufficient length and store it in a secure place (i.e. a password manager). 
+            4. To set up 2FA, type in ‘google-authenticator’
+                1. Answer ‘y’ to all questions asked during the setup
+                2. Save the secret key, verification code, and scratch codes in a safe place. These are all for your user and can be used to login or to recover as needed.
+            5. Install Google Authenticator, Duo, Authy, or other google-authenticator compatible app on your phone or device.
+            6. On your 2FA phone app, add an account, and then scan the barcode or enter the 16 character secret key from step 4’s output.
+            7. Log out and then log back in to check and make sure it worked!
+    5. All of your secondary admin users should be setup now.
+30. You can now begin the Indy Node installation using the Validator Preparation Guide.
```

Comment:
Is there a link available to the guide?

Reply:
not yet, a future PR will have the guide in it